### PR TITLE
Update common-pixracer-overview.rst

### PR DESCRIPTION
**Pull request by kd0aij**
add info on FrSky telemetry port

### CODE REVIEW

File: common/source/docs/common-pixracer-overview.rst [hunk 1]
```diff
@@ -73,6 +73,7 @@ Specifications
    -  TELEM1/TELEM2
    -  Wifi serial
    -  FrSky Telemetry serial
+   Note: The FrSky telemetry port is implemented using a 74LVC2G86 NOR gate as an inverter
```
Comment (Hwurzburg) [inline common/source/docs/common-pixracer-overview.rst:76]:
```suggestion
   Note: The FrSky telemetry port is implemented using a 74LVC2G86 NOR gate as an inverter to both the RX and TX lines
```

File: common/source/docs/common-pixracer-overview.rst [hunk 2]
```diff
@@ -73,6 +73,7 @@ Specifications
    -  TELEM1/TELEM2
    -  Wifi serial
    -  FrSky Telemetry serial
+   Note: The FrSky telemetry port is implemented using a 74LVC2G86 NOR gate as an inverter to both the RX and TX lines
```
Comment (kd0aij) [inline common/source/docs/common-pixracer-overview.rst:76]:
```suggestion
   Note: The FrSky telemetry port is implemented using a 74LVC2G86 NOR gate as a programmable inverter to both the RX and TX lines
```

Comment (Hwurzburg) [reply common/source/docs/common-pixracer-overview.rst]: actually this opens a small can of worms...can we chat when you get a chance?

**Comment by Hwurzburg** [comment]
thanks...added a bit....merging...

**Pull request merged**